docs/reference/release-notes/7.7.asciidoc

```diff
@@ -1,22 +1,612 @@
+* Supplement input checks on received request parameters {pull}52229[#52229]
+* Fix issue with timezone when paginating {pull}52101[#52101] (issue: {issue}51258[#51258])
+* Fix ORDER BY on aggregates and GROUPed BY fields {pull}51894[#51894] (issue: {issue}50355[#50355])
+* Fix milliseconds handling in intervals {pull}51675[#51675] (issue: {issue}41635[#41635])
+* Fix ORDER BY YEAR() function {pull}51562[#51562] (issue: {issue}51224[#51224])
+* Change the way unsupported data types fields are handled {pull}50823[#50823]
+* Selecting a literal from grouped by query generates error {pull}41964[#41964] (issues: {issue}41413[#41413], {issue}41951[#41951])
+
+Search::
+* Fix Term Vectors with artificial docs and keyword fields {pull}53504[#53504] (issue: {issue}53494[#53494])
+* Fix concurrent requests race over scroll context limit {pull}53449[#53449]
+* Fix pre-sorting of shards in the can_match phase {pull}53397[#53397]
+* Fix potential NPE in FuzzyTermsEnum {pull}53231[#53231] (issue: {issue}52894[#52894])
+* Fix inaccurate total hit count in _search/template api {pull}53155[#53155] (issue: {issue}52801[#52801])
+* Harden search context id {pull}53143[#53143]
+* A regression that allowed negative scores in a script_score query was corrected.
+  A behaviour is restored that throws an error if script_score query produces
+  a negative score. {pull}52478[#52478] (issue: {issue}48465[#48465])
+
+Snapshot/Restore::
+* Fix Non-Verbose Snapshot List Missing Empty Snapshots {pull}52433[#52433]
+* Fix Inconsistent Shard Failure Count in Failed Snapshots {pull}51416[#51416] (issue: {issue}47550[#47550])
+* Fix Overly Aggressive Request DeDuplication {pull}51270[#51270] (issue: {issue}51253[#51253])
+
+Store::
+* Fix synchronization in ByteSizeCachingDirectory {pull}52512[#52512]
+
+Transform::
+* Fixing naming in HLRC and _cat to match API content {pull}54300[#54300] (issue: {issue}53946[#53946])
+* Transform optmize date histogram {pull}54068[#54068] (issue: {issue}54254[#54254])
+* Add version guards around Transform hidden index settings {pull}54036[#54036] (issue: {issue}53931[#53931])
+* Fix NPE in derive stats if shouldStopAtNextCheckpoint is set {pull}52940[#52940]
+* Fix mapping deduction for scaled_float {pull}51990[#51990] (issue: {issue}51780[#51780])
+* Fix stats can return old state information if security is enabled {pull}51732[#51732] (issue: {issue}51728[#51728])
+
+
+
+[[upgrade-7.7.0]]
+[float]
+=== Upgrades
+
+Authentication::
+* Update oauth2-oidc-sdk to 7.0 {pull}52489[#52489] (issue: {issue}48409[#48409])
+
+Engine::
+* Upgrade to lucene 8.5.0 release {pull}54077[#54077]
+* Upgrade to final lucene 8.5.0 snapshot {pull}53293[#53293]
+* Upgrade to Lucene 8.5.0-snapshot-c4475920b08 {pull}52950[#52950]
+
+Features/Features::
+* Upgrade Tika to 1.23 {pull}52402[#52402]
+
+Features/Ingest::
+* Upgrade Tika to 1.24 {pull}54130[#54130] (issue: {issue}52402[#52402])
```

Comment:
These two should be in the same category but were just mislabelled.
```suggestion
Features/Ingest::
* Upgrade Tika to 1.23 {pull}52402[#52402]
* Upgrade Tika to 1.24 {pull}54130[#54130] (issue: {issue}52402[#52402])
```

Reply:
Thanks. Actually #52402 was closed (no merge), so I took the liberty to edit your suggestion for ease of committing it here.